server/prisma/schema.prisma

```diff
@@ -10,26 +10,95 @@ datasource db {
 
 // This comes with the boilerplate.
 model User {
-  id                  String          @id @default(cuid())
-  createdAt                        DateTime        @default(now())
-  updatedAt                        DateTime        @updatedAt
-  email           String  @unique
-  password      String
-  role          String
-  lastLogin     DateTime?  
-  // LL: https://www.smashingmagazine.com/2017/11/safe-password-resets-with-json-web-tokens/
-  resetPasswordToken String
+  id                       String             @id @default(cuid())
+  createdAt                DateTime           @default(now())
+  updatedAt                DateTime           @updatedAt
+  email                    String             @unique
+  password                 String
+  role                     String
+  lastLogin                DateTime?
+  resetPasswordToken       String
   dateResetPasswordRequest DateTime?
-  validateEmailToken String
-  isEmailValidated Boolean
-  name  String
+  validateEmailToken       String
+  isEmailValidated         Boolean
+  name                     String
+  feeds                    Feed[] // one to many
+  articles                 ArticlesOnUsers[]
+  settings                 Settings?
+  subscriptionPlans        SubscriptionPlan[]
 }
 
 model Feed {
+  id        String               @id @default(cuid())
+  title     String
+  createdAt DateTime             @default(now())
+  updatedAt DateTime             @updatedAt
+  deletedAt DateTime? //admin could delete
+  user      User                 @relation(fields: [userId], references: [id])
+  userId    String
+  sources   SourceFeedRelation[] // one to many
+}
+
+model SourceFeedRelation {
+  feed     Feed   @relation(fields: [feedId], references: [id])
+  feedId   String
+  source   Source @relation(fields: [sourceId], references: [id])
+  sourceId String
+
+  @@id([feedId, sourceId])
 }
 
 model Source {
+  id        String               @id @default(cuid())
+  createdAt DateTime             @default(now())
+  updatedAt DateTime             @updatedAt
+  slug      String //url
+  title     String
+  deletedAt DateTime? //admin could delete
+  Article   Article[]
+  feeds     SourceFeedRelation[]
 }
 
 model Article {
+  id           String            @id @default(cuid())
+  createdAt    DateTime          @default(now())
+  updatedAt    DateTime          @updatedAt
+  deletedAt    DateTime? //admin could delete
+  source       Source            @relation(fields: [sourceId], references: [id])
+  sourceId     String
+  author       String?
+  title        String
+  slug         String //url
+  shortContent String
+  users        ArticlesOnUsers[]
```

Comment:
this field should not be called users, it should be called "userInteractions" or something like this. as this is not a `User[]` type.